app/models/sushi.rb

```diff
@@ -158,33 +158,21 @@ module MetricTypeCoercion
     included do
       attr_reader :metric_types, :metric_type_in_params
     end
-    ALLOWED_METRIC_TYPES = [
```

Comment:
the list of allowed types differs per report (even if we aren't accounting for each of them right now). so I moved the const into the respective reports so`#coerce_metric_types` is still reusable.